### PR TITLE
[docs][cert-manager] Add documentation about creating DNS-Challenge based cert-manager issuer

### PR DESCRIPTION
**Pull request by Suselz**
## Description
Added block in the documentation that illustrates how to create a cert-manager issuer that uses DNS Challenge. It also provides necessary context to help users understand scenarios where they might need such an issuer.

## Why do we need it, and what problem does it solve?

This information answers some typical questions that may arise during the use of the module.

## What is the expected result?

The quality of documentation will improve

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: fix 
summary: Add documentation about creating DNS-Challenge based cert-manager issuer
impact_level:  low
```

### CODE REVIEW

File: modules/101-cert-manager/docs/FAQ_RU.md [hunk 1]
```diff
@@ -58,6 +58,71 @@ default            example-com                     13m
...
+У `сert-manager` есть поддержка механизмов для создания TXT-записей в некоторых популярных DNS: AzureDNS, Cloudflare, Google Cloud DNS, и другие. Полный перечень доступен [в документации cert-manager](https://cert-manager.io/docs/configuration/acme/dns01/)
+
+Пример использования AWS Route53 доступен в разделе [`Как защитить учетные данные cert-manager`](#как-защитить-учетные-данные-cert-manager)
+
+Использование сторонних DNS-провайдеров реализуется через метод `webhook`.  Когда cert-manager выполняет вызов `ACME` `DNS01`, он отправляет вызов на webhook-сервер, который затем выполняет нужные операции для обновления записи DNS.
+При использовании данного метода требуется разместить сервис, который будет обрабатывать хук и осуществлять создание TXT записи в DNS-провайдере.
+
+В качестве примера рассмотрим использование сервиса Yandex Cloud DNS
+
+Для обработки webhook предварительно разместим в кластере сервис `Yandex Cloud DNS ACME webhook` согласно [официальной документации](https://github.com/yandex-cloud/cert-manager-webhook-yandex)
```
Comment (nabokihms) [inline modules/101-cert-manager/docs/FAQ_RU.md:93]: I think the part about YC is good enough and we do not need to dive into all the nuances of installation of the webhook.

**Comment by z9r5** [comment]
@Suselz Please answer Maxim's comments. If you resolve the comments, add something like "Done", "Fixed" etc.

File: modules/101-cert-manager/docs/FAQ_RU.md [hunk 2]
```diff
@@ -58,6 +58,71 @@ default            example-com                     13m
 * selfsigned
 * selfsigned-no-trust
 
+Если требуется поддержка других типов сертификатов, то вы можете добавить их самостоятельно.
+
+## Как добавить дополнительный `ClusterIssuer`?
+
```
[the remark this answers is not in the document]
Comment (Suselz) [reply modules/101-cert-manager/docs/FAQ_RU.md]: Moved section as recommended

File: modules/101-cert-manager/docs/FAQ_RU.md [hunk 3]
```diff
@@ -58,6 +58,71 @@ default            example-com                     13m
...
+### В каких случаях требуется дополнительный `ClusterIssuer`
+
+В стандартной поставке присутствуют два `ClusterIssuer`, которые издают сертификаты из доверенного публичного удостоверяющего центра LetsEncrypt - `letsencrypt` и `letsencrypt-staging`.
```
[the remark this answers is not in the document]
Comment (Suselz) [reply modules/101-cert-manager/docs/FAQ_RU.md]: Based on this information, I globally completed the documentation.

File: modules/101-cert-manager/docs/FAQ_RU.md [hunk 4]
```diff
@@ -58,6 +58,71 @@ default            example-com                     13m
...
+Чтобы издать сертификаты на доменное имя через LetsEncrypt, сервис требует осуществить подтверждение владения доменом. 
+Cert-manager поддерживает несколько методов для подтверждения владения доменом при использовании `ACME`(Automated Certificate Management Environment):
+* `HTTP-01` -  метод подразумевает, что cert-manager создаст временный Pod в кластере, который будет слушать на определенном URL для подтверждения владения доменом. Для его работы необходимо иметь возможность направлять внешний трафик на этот Pod, обычно через `Ingress`.
+* `DNS-01` -  метод подразумевает, что cert-manager делает запись TXT запись в DNS для подтверждения владения доменом. У cert-manager есть встроенная поддержка популярных провайдеров DNS: AWS Route53, Google Cloud DNS, Cloudflare, и другие.
+
+`ClusterIssuers` присутствующие в поставке используют метод `HTTP-01`.
```
[the remark this answers is not in the document]
Comment (Suselz) [reply modules/101-cert-manager/docs/FAQ_RU.md]: Based on this information, I globally completed the documentation.

File: modules/101-cert-manager/docs/FAQ_RU.md [hunk 5]
```diff
@@ -58,6 +58,71 @@ default            example-com                     13m
...
+
+Таким образом дополнительный `ClusterIssuer` может потребоваться в следующих случаях:
+1. Требуется издание сертификатов в удостоверяющем центре, отличным от LetsEncrypt (в т.ч. в приватном). Поддерживаемые УЦ доступны [в документации cert-manager](https://cert-manager.io/docs/configuration/)
+2. Требуется создание сертификата через LetsEncrypt, но через метод `DNS-01`
+   1. Требуется издание wildcard сертификатов (доступно только через метод `DNS-01`)
```
[the remark this answers is not in the document]
Comment (Suselz) [reply modules/101-cert-manager/docs/FAQ_RU.md]: Rewrote some of the documentation. No longer relevant.

**Comment by Suselz** [comment]
@z9r5, replied to all comments

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Review by Lada7878 — APPROVED** [review]

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [11357](https://github.com/deckhouse/deckhouse/pull/11357) to the branch [release-1.67](https://github.com/deckhouse/deckhouse/tree/release-1.67) successful!

**Pull request merged**